Makefile

```diff
@@ -1,74 +0,0 @@
-# referenced from https://github.com/notaryproject/notation/blob/main/Makefile
```

Comment:
Can you please either add this file back OR add instructions on how to build the binary?  

Great that we have the GitHub testing/release/build/lint/etc, but developers need to know how to locally run what is running in CI.

Reply:
Done